.github/workflows/ci.yml

```diff
@@ -179,19 +180,31 @@ jobs:
       - name: Install scarb
         uses: software-mansion/setup-scarb@v1
         with:
-          scarb-version: "2.6.4"
+          scarb-version: "2.7.0"
       - name: Install deps
         run: make deps
       - name: Build cairo-native-runtime library.
-        run: cargo build --profile=ci --package=cairo-native-runtime
+        run: export PATH=$HOME/.rustup/toolchains/stable-aarch64-apple-darwin/bin:$PATH && cargo build --profile=ci --package=cairo-native-runtime
       - name: Run tests
-        run: make test-ci
+        run: export PATH=$HOME/.rustup/toolchains/stable-aarch64-apple-darwin/bin:$PATH && make test-ci
       - name: test-cairo
-        run: make test-cairo
+        run: export PATH=$HOME/.rustup/toolchains/stable-aarch64-apple-darwin/bin:$PATH && make test-cairo
```

Comment:
Does this workaround still apply? If not, we should create an issue to remove it later.